mteb/models/repllama_models.py

```diff
@@ -144,6 +144,15 @@ def loader_inner(**kwargs: Any) -> Encoder:
     open_source=True,
     revision="01c7f73d771dfac7d292323805ebc428287df4f9-6097554dfe6e7d93e92f55010b678bcca1e233a8",  # base-peft revision
     release_date="2023-10-11",
+    n_parameters=7_000_000,
+    memory_usage=None,
+    max_tokens=None,
+    embed_dim=None,
+    license="apache-2.0",
+    reference="https://huggingface.co/samaya-ai/castorini/repllama-v1-7b-lora-passage",
+    similarity_fn_name="cosine",
+    framework=["PyTorch"],
+    use_instuctions=True,
```

Comment:
No instruction for this one, although it has the query prefix and passage prefix (do those count?)

And 4096 embedding and max tokens. Also tevatron training code with Pytorch.  Same for both of the RepLlama models.

Reply:
added

Reply:
> No instruction for this one, although it has the query prefix and passage prefix (do those count?)

I have counted it as an instruction yes